src/components/chart/styles/ChartTooltipContent.scss

```diff
@@ -0,0 +1,20 @@
+@import '~coveo-styleguide/scss/common/palette';
+@import '~coveo-styleguide/scss/variables';
+
+$chart-tooltip-content-size: 3px;
+
+.chartTooltipContent {
+  font-size: 0.75rem;
```

Comment:
should be in the vapor as variable IMO

Reply:
I'll change for $small-font-size which is 1px bigger